game/src/js/objects/Character.js

```diff
@@ -200,4 +200,54 @@ export default class Character extends Phaser.Physics.Arcade.Sprite {
       character.destroy();
     }
   }
-}
\ No newline at end of file
+  
+  // sets the character to use an AI, stalking and aiming at the player.
+  // doesn't deal with multiple opponents yet.
+  setAI(player) {
+    this.AIOn = true;
+    this.targetPlayer = player;
+
+  }
+
+  // turns off the current AI.
+  turnAIOff() {
+    this.AIOn = false;
+    this.targetPlayer = null;
+  }
+
+  // performs one tick worth of time of what the AI is going to do.
+  updateAI() {
+    if (this.isDead || !this.AIOn) {
+      return;
+    }
+    const targetXPosition = this.targetPlayer.x;
+    const targetYPosition = this.targetPlayer.y;
+    const xDifference = this.x - targetXPosition;
+    const yDifference = this.y - targetYPosition;
+    const distance = Math.sqrt(xDifference * xDifference + yDifference * yDifference);
+    if (distance > 300) {
+      return;
+    }
+    var xChange = 0;
+    var yChange = 0;
+    if (xDifference > 0) {
+      xChange = -10;
+    } else if (xDifference < 0) {
+      xChange = 10;
+    }
+    if (yDifference > 0) {
+      yChange = -10;
+    } else if (yDifference < 0) {
+      yChange = 10;
+    }
+    const vector = [xChange, yChange];
+    this.setMotion(vector);
```

Comment:
Here's the problem with the vector. This is passing in an array instead of a Phaser.Math.Vector2